### ISSUE TITLE
Question: Does sqlitebrowser support 3d plot 

### ISSUE THREAD

**Issue by tekmanBlade** [sqlitebrowser/sqlitebrowser#397]
I found it support 2d chart on GUI. Does it support 3d plt based on X, Y, Z data?

**Comment by justinclift** [comment]
I don't think so yet, but it's a decent idea if someone wants to pick up the task and write the code.

Are you any good with C++? :smile:

**Comment by tekmanBlade** [comment]
Thanks. I'll have have a try.

**Comment by justinclift** [comment]
Cool. :smile:

**Comment by MKleusberg** [comment]
Unfortunately this is going to be quite difficult :frowning: The reason for this is that we rely on the QCustomPlot library for plotting which itself only support 2D plots. So you're left with three options as far as I can tell:
* Finding a new open-source library to replace QCustomPlot and rewrite the sqlitebrowser to use that instead (I don't know of any such library...)
* Extending QCustomPlot and using the new features in sqlitebrowser (never looked into their source code, might be easy to to, might be next to impossible)
* Trying to fake it using colour gradients or what not (this might be interesting but probably will be very difficult to get right if it works at all)

**Comment by tekmanBlade** [comment]
Thanks for the reminder.

**Comment by mastras23** [comment]
https://alandefreitas.github.io/matplotplusplus ?
